e2e/provision/Vagrantfile

```diff
@@ -20,6 +20,9 @@ Vagrant.configure('2') do |config|
   config.vm.box_check_update = false
   config.vm.synced_folder './', '/vagrant'
 
+  config.vm.network 'forwarded_port', guest: 7007, guest_ip: '127.0.0.1', host: 7007
+  config.vm.network 'forwarded_port', guest: 3000, guest_ip: '172.18.0.200', host: 3000
```

Comment:
In generatl, might be a good idea to open a separate PR when adding other functionalities.